ruff.toml

```diff
@@ -70,7 +70,15 @@ ignore = [
   "N817", # imported as acryonym
   "B023", # doesn't bind loop var, we do this a lot in torch
   "D100", # module-level docstrings
+  "D417", # documentation for every function parameter.
+  "N806", # ignore uppercased variables
+  "N812", # import as uppercased
+  "N803", # lowercased args
   "N805", # first param needs to be self; pydantic breaks this sometimes
+  "D415", # First line should end with a period, question mark, or exclamation point
```

Comment:
added some of these to bypass numerous changes on import from lm-buddy, cross-checked with the ruff file there as well.